recipes/leftbar/callbox/callbox_default.story.vue

```diff
@@ -66,39 +68,11 @@
         </dt-button>
       </div>
     </template>
-    <template #bottom>
-      <dt-item-layout>
-        <template #left>
-          <dt-button
-            circle
-            importance="clear"
-          >
-            <template #icon>
-              <dt-icon
-                name="share-screen"
-                size="400"
-              />
-            </template>
-          </dt-button>
-        </template>
-        <template #default>
-          <span class="d-fs-100">Sharing screen</span>
-        </template>
-        <template #right>
-          <dt-button
-            importance="clear"
-            class="d-fc-critical"
-          >
-            <template #icon>
-              <dt-icon
-                name="stop-filled"
-                size="200"
-              />
-            </template>
-            Stop
-          </dt-button>
-        </template>
-      </dt-item-layout>
+    <template
+      v-if="bottom"
+      #bottom
+    >
+      <div v-html="bottom" />
```

Comment:
I think it is a bit unclear just looking at the default story what is included as part of the component and what is not. For example it seems like the AI and hang up buttons are part of this component but they actually are not you can pass anything into the slot.

I think we should for the default example have a string that just says "right slot", and then under the variants story we can show more defined examples like showing the AI / hangup button and such. Maybe a few different variants from the design examples here. This might be relevant for the bottom slot too as the icon is not included.

Reply:
That makes sense! I'll update the examples